group/templates/group/editAnnounce.html

```diff
@@ -0,0 +1,38 @@
+{% extends "index/base.html" %}
+{% load static %}
+{% block title_name %}
+<title>Edit for Announce...</title>
+{% endblock title_name %}
+{% block import_source %}
+<script src="{%static 'group/js/editAnnounce.js'%}"></script>
+{% endblock import_source %}
+{% load bootstrap %}
+{% block body_block %}
+<br>
+<br>
+<div class="col-lg-offset-1 col-lg-10">
+  <form action="." method="POST">
+    {% csrf_token %}
+    <div class="alert alert-dismissable alert-info">
+      <button type="button" class="close" data-dismiss="alert">×</button>
+      <h4>Hint: <span style="color:red">Title</span> is required!</h4>
+    </div>
+    {% if user_is_owner or user_is_coowner or user_has_admin_auth %}
```

Comment:
A coowner should not have auth to delete group.
BTW, your view didn't pass ```user_has_admin_auth``` to template (because it's included in ```user_is_owner```)